### PR TITLE
feat: add avatar & badge components

### PR DESCRIPTION
**Pull request by irfan-nawaz**
added avatars of different sizes and created avatar badges and icon badges

### CODE REVIEW

File: Components/Badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
+@import url("../../main.css");
+@import url('../../colors.css');
+
+.badge-conatiner {
+  display: flex;
+  flex-direction: row;
```
Comment (shaik-saad) [inline Components/Badge/badge.css]: You can remove flex-direction, as by default it is row only.

File: Components/Badge/badge.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+}
+
+/* in-stock colors */
+
+.colors {
+  display: flex;
+  position: relative;
+  display: inline;
+
+}
```
Comment (shaik-saad) [inline Components/Badge/badge.css]: Here you can either keep flex or inline, only the display: inline will work here as it comes later in the document. If you wanted to use flex but did not want create a new block then you can use 'display: inline-flex'.

**Review by shaik-saad — CHANGES_REQUESTED** [review]
Added some suggestions, check them out

**Review by shaik-saad — APPROVED** [review]
You're good to merge now

**Comment by irfan-nawaz** [comment]
> You're good to merge now

Thanks, man for your review really appreciate it 👍🏼

File: index.html [hunk 3]
```diff
@@ -317,6 +319,73 @@ <h2>Nike Air Max</h2>
...
+                mail
+            </span>
+            <sup class="badge-icon">4</sup>
+        </div>
+    </div>
+
+    <div class="status">
+        <img class="avatar round sm"
+            src="https://images.unsplash.com/photo-1561409106-fece1abb71cb?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80"
+            alt="">
```
Comment (ruchiragoswami) [inline index.html]: A good practice is to put the 'alt txt'. Even for avatar, I personally think there should be something.

File: index.html [hunk 4]
```diff
@@ -317,6 +319,73 @@ <h2>Nike Air Max</h2>
...
+        <div class="dnd"></div>
+    </div>
+    <div class="status">
+        <img class="avatar round sm"
+            src="https://images.unsplash.com/photo-1561409106-fece1abb71cb?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80"
+            alt="">
+        <div class="offline"></div>
```
Comment (ruchiragoswami) [inline index.html:387]: I am sure these empty divs are there for some reason!

File: index.html [hunk 5]
```diff
@@ -13,6 +13,8 @@
     <link rel="stylesheet" href="./Components/Card/card.css">
     <link rel="stylesheet" href="./Components/Input/input.css">
     <link rel="stylesheet" href="./Components/Image/image.css">
+    <link rel="stylesheet" href="./Components/Avatar/avatar.css">
+    <link rel="stylesheet" href="./Components/Badge/badge.css">
```
Comment (ruchiragoswami) [inline index.html]: I personally prefer to use all lower case letters in the website files because that is considered a good practice.

**Review by ruchiragoswami — COMMENTED** [review]
Great work Irfan! I am sharing my thoughts with you as comments, not asking you to change anything. 
With new rules in effect  about reviewing a PR, I am not sure if I can approve this. But overall, good work :)

File: index.html (hunk 4 above)
Comment (irfan-nawaz) [reply index.html:387]: yes, it is there to create a status for avatars.

File: index.html (hunk 5 above)
Comment (irfan-nawaz) [reply index.html]: oh, I just read about it online, and thanks for telling me 🙌🏼

File: index.html (hunk 3 above)
Comment (irfan-nawaz) [reply index.html]: I have added them now going to commit changes 👍🏼

**Review by ruchiragoswami — APPROVED** [review]
Thanks for implementing my suggestions :)

**Pull request merged**